lib/gladiator.rb

```diff
@@ -1,2 +1,18 @@
 class Gladiator
+  attr_accessor :name, :weapon
+
+  def initialize (name, weapon)
+    @name = name
+    @weapon = weapon
+  end
+  #
+  # def gladiator (@name, weapon)
+  #
+  # end
```

Comment:
A gladiator method, that accepts a name and weapon, on a Gladiator?  Arguments as instance vars.  Glad this is commented out. :)